.size-snapshot.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file should be added to `.gitignore` as it is generated, so it should not be committed.

Reply:
The idea is to commit size snapshot so on every new commit we could see impact on final size.